canonical-webteam/practices
# Make into a Jekyll site

Here I've added basic scaffolding to display these practices as a Jekyll site. When this is merged, this should be displayed with GitHub pages at https://canonical-webteam.github.io/practices/.

I copied the markup structure & CSS from https://docs.vanillaframework.io/, which is the best example we have of an official site with a sidebar nav (as far as I'm aware).

I've been quite quick and dirty here, as I don't want to spend too long on this - any solution as all feels like an improvement. So I:

- Simply copied the CSS from that site directly, as I can't be arsed to set up a whole _sass building chain (which is problematic with GitHub pages anyway).
- Lazily used https://assets.ubuntu.com/v1/84489620-picto-knowledge-orange.svg as the logo, and hacked an `<h4>` to give the site a title next to the logo

QA
--

See https://nottrobin.github.io/practices/.

If you want to test it locally, you'll need to `sudo gem install jekyll`, then run `jekyll serve`.
